static/app/views/issueDetails/groupSimilarIssues/similarStackTrace/toolbar.tsx

```diff
@@ -1,24 +1,52 @@
-import {Component} from 'react';
+import React, {Component} from 'react';
 import styled from '@emotion/styled';
 
+import {addSuccessMessage} from 'sentry/actionCreators/indicator';
 import {Button} from 'sentry/components/button';
 import Confirm from 'sentry/components/confirm';
 import PanelHeader from 'sentry/components/panels/panelHeader';
 import ToolbarHeader from 'sentry/components/toolbarHeader';
 import {t} from 'sentry/locale';
 import GroupingStore from 'sentry/stores/groupingStore';
 import {space} from 'sentry/styles/space';
+import {Group, Organization} from 'sentry/types';
+import {trackAnalytics} from 'sentry/utils/analytics';
 
 type Props = {
   onMerge: () => void;
+  organization?: Organization;
+  parentGroupId?: number;
 };
 
 const initialState = {
   mergeCount: 0,
+  mergeList: [],
```

Comment:
because type State is using a goofy typeof initialState, you'll need to cast mergeList to something
```suggestion
  mergeList: [] as Something[],
```